Wikipedia/Code/AppDelegate.m

```diff
@@ -114,6 +113,7 @@ - (void)applicationWillEnterForeground:(UIApplication *)application {
 - (void)applicationDidBecomeActive:(UIApplication *)application {
     // Restart any tasks that were paused (or not yet started) while the application was inactive. If the application was previously in the background, optionally refresh the user interface.
     [[NSUserDefaults wmf_userDefaults] wmf_setAppBecomeActiveDate:[NSDate date]];
+    [self resumeAppIfNecessary];
```

Comment:
Resume is now always delayed until at least `applicationDidBecomeActive:` - this way the app isn't fully resumed for a background fetch